### PR TITLE
chore(*): update polkadot versions to match SDK v22 versions

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

update polkadot versions to match SDK v22 versions

### Breaking Changes

None

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v1
       - uses: actions/setup-node@v1
```
Comment (VictorVicente) [inline .github/workflows/main.yml:15]: Should we use v3 as other projects?

**Review by VictorVicente — COMMENTED** [review]
There is a redundat `peerDependencies` at `hashicorp-vault-signing-manager`:
```
"peerDependencies": {
    "@polymeshassociation/polymesh-sdk": ">=15.0.0",
    "@polymeshassociation/signing-manager-types": "^1.0.0"
  }
```
Is it safe to remove it?

**Comment by prashantasdeveloper** [comment]
Also, marking the commits as chore/ci won't release a new version

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**